_plugins/version.rb

```diff
@@ -0,0 +1,52 @@
+# Aggregate the En lesson versions together
+Jekyll::Hooks.register :site, :post_read do |site|
```

Comment:
@nscyclone I want to incorporate your indicator idea and better messaging and get this merged but I think the change count may be a future enhancement, how's that sound?